plugins/wazo_aastra/v6_4_0_SP2/entry.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2015-2024 The Wazo Authors  (see the AUTHORS file)
```

Comment:
why 2015?

Reply:
Because it's exactly the same code as the other plugins